app/src/main/java/org/schabi/newpipe/util/ShareUtils.java

```diff
@@ -21,80 +21,87 @@ private ShareUtils() {
     /**
      * Open an Intent to install an app.
      * <p>
-     * This method will first try open to Google Play Store with the market scheme and falls back to
-     * Google Play Store web url if this first cannot be found.
+     * This method will first try to open the Google Play Store with the market scheme and falls
+     * back to Google Play Store web url if this first cannot be found.
```

Comment:
That JDoc is wrong We try to open a store with that. The Google Play Store is not the only store listening for that scheme, e.g. F-Droid and Aurora also accept that. The method allows the user to choose the store if no default store is set.

Reply:
Oh yes, sorry. I will update the JavaDoc.